set.mm

```diff
@@ -775924,6 +775930,46 @@ coordinates of the intersection points of a (nondegenerate) line and a
       ( wa wi impexp 3bitr3g ) ABCIDJEFIGJBCDJJEFGJJHBCDKEFGKL $.
   $}
 
+  ${
+    monepilem.1 $e |- ( ph -> ( ps <-> ( ch /\ th ) ) ) $.
+    monepilem.2 $e |- ( ( ph /\ ch ) -> th ) $.
+    $( Common lemmas for proving monomorphisms, epimorphisms, and potentially
+       others.  (Contributed by Zhi Wang, 24-Sep-2024.) $)
+    monepilem $p |- ( ph -> ( ps <-> ch ) ) $=
+      ( simprbda ex wa ancld sylibrd impbid ) ABCABCABCDEGHACCDIBACDACDFHJEKL
+      $.
```

Comment:
`mpbiran3d` is a better label, with comment, for instance:
> ($ Equivalence with a conjunction one of whose conjuncts is a consequence of the other.  Deduction form.

Reply:
https://github.com/metamath/set.mm/pull/4248/commits/e1ac292af60d2603dec73e54b836a8e55c61b3b1